### DISCUSSION TITLE
Add pattern for goNext and goPrevious for drive.google.com

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1078] [Q&A]
Think should cover what I'm trying to do:
`n@drive\.google\.com/drive/u/(\d)@drive.google.com/drive/u/$${$1/0:1:1}@`

Essentially, switching between accounts using `[[` and `]]`
This works but for some reason I can't go back from `u/1` to `u/0`

**Comment by gdh1995** [comment]
Sorry there're quite a few math bugs when I wrote this feature.
- one is "min value is at least `1`"
- another is it can not go from Page 9th to Page 10th...

I'll fix it in Vimium C v2.0+. But, now v1.99 may only get patches for severe problems, so this bug will be left there.
